training/css/training.css

```diff
@@ -0,0 +1,5 @@
+#table-test-login {
```

Comment:
Use 2 spaces for each level of indentation, the same standard as Drupal’s PHP and JavaScript code.
https://www.drupal.org/node/1887862